etcdserver/etcdserverpb/rpc.proto

```diff
@@ -267,6 +267,10 @@ message WatchCreateRequest {
   bytes range_end = 2;
   // start_revision is an optional revision (including) to watch from. No start_revision is "now".
   int64 start_revision = 3;
+  // ProgressReport 'true' subscribes to periodic watch status updates.
+  bool progress_report = 4;
+  // ReportInterval is progress report frequency in seconds. If 5, it notifies every 5 seconds.
```

Comment:
report frequency should be at minute level, not second level i think. say we have 100K watchers. 5 second per watch = 20k request / second. also we usually keep hours history, minute level reporting is already more than enough.